scripts/scil_compute_seed_density_map.py

```diff
@@ -58,10 +60,18 @@ def main():
     # Can handle streamlines outside of bbox
     sft = load_tractogram(args.tractogram_filename, 'same',
                           bbox_valid_check=False)
-    # Streamlines are saved in RASMM but seeds are saved in VOX
-    # This might produce weird behavior with non-iso
+    # Streamlines are saved in RASMM by nibabel but seeds are saved in VOX,
+    # the default space in dipy.
+    # Also, origin should be center when creating the seeds (wee below, we
+    # are using round, not floor; works with center origin), which is the
+    # default in dipy.
+    # Changing sft here just to make sure, but it will not have consequences on
```

Comment:
If you can validate that this indeed doesn't change anything I think we should just remove to_vox and to_center below.

Reply:
Ok, I'll check that soon.

Reply:
I don't know why they are there, they do not modify anything related to seed. They should be removed.